### ISSUE TITLE
TypeError: Object.keys called on non-object

### ISSUE THREAD

**Issue by 81735595** [npm/gauge#88]
run in node 0.12.7，gives me:
`
/Users/zhengxingcheng/data/lab/gauge/index.js:88
  if (theme && (Object.keys(theme).length === 0 || theme.hasUnicode != null ||
                       ^
TypeError: Object.keys called on non-object
    at Function.keys (native)
    at Object.Gauge._computeTheme (/Users/zhengxingcheng/data/lab/gauge/index.js:88:24)
    at Object.Gauge (/Users/zhengxingcheng/data/lab/gauge/index.js:51:20)
    at nextBar (/Users/zhengxingcheng/data/lab/gauge/demo.js:19:12)
    at Object.<anonymous> (/Users/zhengxingcheng/data/lab/gauge/demo.js:13:1)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
`

**Comment by 81735595** [comment]
```
/Users/zhengxingcheng/data/lab/gauge/index.js:88
  if (theme && (Object.keys(theme).length === 0 || theme.hasUnicode != null ||
                       ^
TypeError: Object.keys called on non-object
    at Function.keys (native)
    at Object.Gauge._computeTheme (/Users/zhengxingcheng/data/lab/gauge/index.js:88:24)
    at Object.Gauge (/Users/zhengxingcheng/data/lab/gauge/index.js:51:20)
    at nextBar (/Users/zhengxingcheng/data/lab/gauge/demo.js:19:12)
    at Object.<anonymous> (/Users/zhengxingcheng/data/lab/gauge/demo.js:13:1)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
```

**Comment by iarna** [comment]
Fixed by #89
